src/Helper/TemplateGuesser.php

```diff
@@ -89,16 +89,22 @@ private function resolve(string $namespace, string $class, string $method): stri
 
         $action = Strings::replace($method, self::ACTION_MATCH_REGEX, '');
 
+        $action = Strings::replace(
+            $action,
+            self::SMALL_LETTER_BIG_LETTER_REGEX,
+            '\\1_\\2'
```

Comment:
this somehow changed to `1_\\2` in scoped rector/rector https://github.com/rectorphp/rector/commit/67791a95820367d299e8e891b18d91afc9fb08d5#diff-02557c06605d556a23eae70c6c7b37cf26cc8f54ca434718889e7a97d8d8a2e3

@MikeAmelung probably changed by php-scoper, could you verify latest rector/rector?

Reply:
I can confirm that it is broken in rectorphp/rector, as that is what initially caught my attention as my controllers were coming out as "Som1_Controller". I thought to mention it in my original pull request comment, but I didn't realize it was a build process issue causing it.

Reply:
any alternative code for it? If no, the scoper.php may need str_replace for this file, eg:

https://github.com/rectorphp/rector-src/blob/1d2d51e20ce11d0f603e42bd8869c1ab6d2a52cc/scoper.php#L82-L89

Reply:
Added a new commit, hopefully this doesn't cause any issues with the scoper? I had copied the '\\1_\\2' from code that was already there, but I feel like '$1_$2' is more common usage with regexes anyway.

https://github.com/MikeAmelung/rector-symfony/commit/1f61d849decfaa1a624482aaf0c16fe065a79e46

Should I create another pull request?

Reply:
Yes please :+1: